MapboxNavigation/NavigationMapView.swift

```diff
@@ -885,64 +870,57 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
         let lineCasing = MGLLineStyleLayer(identifier: identifier, source: source)
         
         // Take the default line width and make it wider for the casing
-        lineCasing.lineWidth = MGLStyleValue(interpolationMode: .exponential,
-                                             cameraStops: routeLineWidthAtZoomLevels.multiplied(by: 0.85),
-                                             options: [.defaultValue : MGLConstantStyleValue<NSNumber>(rawValue: 1.5)])
+        lineCasing.lineWidth = NSExpression(format: "FUNCTION($zoomLevel, 'mgl_interpolateWithCurveType:parameters:stops:', 'linear', nil, %@)", argumentArray: [routeLineWidthAtZoomLevels.multiplied(by: 0.85)])
         
-        lineCasing.lineColor = MGLStyleValue(rawValue: routeAlternateColor)
-        lineCasing.lineCap = MGLStyleValue(rawValue: NSValue(mglLineCap: .round))
-        lineCasing.lineJoin = MGLStyleValue(rawValue: NSValue(mglLineJoin: .round))
-        lineCasing.lineOpacity = MGLStyleValue(rawValue: 0.9)
+        lineCasing.lineColor = NSExpression(forConstantValue: routeAlternateColor)
+        lineCasing.lineCap = NSExpression(forConstantValue: NSValue(mglLineCap: .round))
```

Comment:
I think you can say something like `MGLLineCap.round as NSValue` instead of calling `NSValue(mglLineCap:)`. `NSExpression(forConstantValue: "round")` and `NSExpression(format: "'round'")` are also options.